resources/sass/components/_map-aside.scss

```diff
@@ -125,26 +178,70 @@
             text-decoration: none;
         }
         &__wrapper {
-            display: flex;
-            flex-direction: column;
+            width: 100%;
+            @media screen and (max-width: 1040px) {
+                display: flex;
+                flex-direction: column;
+                justify-content: space-between;
+            }
         }
         &__info {
             display: flex;
             justify-content: space-between;
+            flex-wrap: nowrap;
+        }
+        &__refer {
+            display: flex;
+            justify-content: space-between;
+            margin-right: 5px;
+        }
+        &__time,
+        &__refer {
             color: $c-gray8;
         }
+        &__refer,
+        &__time {
+            @media screen and (max-width: 1040px) {
+                display: none;
+            }
+        }
+        &__btn {
+            display: none;
+            color: $c-red2;
+            width: 100%;
+            min-width: max-content;
```

Comment:
did you test it on ie? or is it not important there?
![image](https://user-images.githubusercontent.com/6440135/65028769-41b71880-d945-11e9-90a6-bbd6f5b69f03.png)

Reply:
+

Reply:
fixed